Documentation/Building.md

```diff
@@ -1,11 +1,64 @@
 # Building and installing Libevent
 ### Jump to:
+- [Prerequisites](#Prerequisites)
 - [Autoconf](#autoconf)
   - [Flags](#autoconf-flags)
 - [Building on Windows](#building-on-windows)
 - [Building on Unix (With CMake)](#building-on-unix-cmake)
 - [CMake Variables](#cmake-variables)
 
+## Prerequisites
+
+### Linux deb-like (ubuntu/debian/...)
+Just install tools using your preferred package manager if you build using autotools:
+```sh
+sudo apt-get install automake autoconf libtool pkg-config
+```
+or build using cmake:
+```sh
+sudo apt-get install cmake
+```
+Doxygen is a tool for generating documentation. Git is used to fetch the package version.  
+Install them if needed:
+```
+sudo apt-get install doxygen git
+```
+It is highly recommended to support OpenSSL encryption:
```

Comment:
Why it is "highly recommended", there can be embedded systems that do not care about this, I would say "libevent has encryption layer, you need openssl for it" (and a note that it is enabled by default, and hence you can have problems if you do not have openssl installed on configure stage)